MEMBER_TRAVEL_FUND.md

```diff
@@ -1,137 +1,101 @@
-  members of @nodejs/community-committee. (People who are members of both
-  committees may only be counted for one or the other. Therefore, any request
-  will always require at least four approvals.)
-* No members of either @nodejs/tsc or @nodejs/community-committee have objected
-  to the request.
-* The pull request has been merged. Any member may merge once other
-  prerequisites have been met.
-
-If a request is blocked by an objection from a member of @nodejs/tsc or
-@nodejs/community-committee, then the only way for the request to be approved is
-for that committee to decide to approve the request using whatever
-decision-making process the  committee uses when consensus is not reached. (For
-example, for the TSC, there is a voting process to make decisions when consensus
-cannot be reached.)
-
-An appointed treasurer from either the @nodejs/tsc or @nodejs/community-committee
-(or both) will liaise with a member of the Node.js Foundation on a regular basis
-to review the status of the travel fund. In the event that any pending requests
-might not be approved because of available funds, a separate issue will be raised.
-Generally speaking, however, members shouldn't need to worry about the status
-of the fund itself.
-
-As examples, purpose of prior sponsored travels include attending TC39 meetings,
-attending [Node.js collaboration summits][], and mentoring [Code and Learn][].
+* The request has approval from at least two members of @nodejs/tsc and two members of @nodejs/community-committee. (People who are members of both committees may only be counted for one or the other. Therefore, any request will always require at least four approvals.)
+* No members of either @nodejs/tsc or @nodejs/community-committee have objected to the request.
+* The pull request has been merged. Any member may merge once other prerequisites have been met.
+
+If a request is blocked by an objection from a member of @nodejs/tsc or @nodejs/community-committee, then the only way for the request to be approved is for that committee to decide to approve the request using whatever decision-making process the committee uses when consensus is not reached. (For example, for the TSC, there is a voting process to make decisions when consensus cannot be reached.)
+
+An appointed treasurer from either the @nodejs/tsc or @nodejs/community-committee (or both) will liaise with a member of the Node.js Foundation on a regular basis to review the status of the travel fund. In the event that any pending requests might not be approved because of available funds, a separate issue will be raised. Generally speaking, however, members shouldn't need to worry about the status of the fund itself.
+
+As examples, purpose of prior sponsored travels include attending TC39 meetings, attending [Node.js collaboration summits](https://github.com/nodejs/summit), and mentoring [Code and Learn](https://github.com/nodejs/code-and-learn).
```

Comment:
purpose of prior sponsored travels include 
->
**the** purpose of prior sponsored travels include**s**